doc/modelling/unit_operations/population_balance_model.rst

```diff
@@ -0,0 +1,154 @@
+.. _pbm_model:
+
+Population balance model (PBM)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The population balance equation is a particle-number continuity equation which describes the evolution of the number density :math:`n` of the particles in the time and space domains. 
+The particles of interest have both internal and external coordinates: the internal coordinate can be chosen as any property of the particles such as the particle size or volume 
+while the external coordinate can be a characteristic dimension of the reactor itself, including the axial position. The current release considers the nucleation, growth and growth rate dispersion. 
+The equations can be solved in CSTR or DPFR (LRM) to model batch or continuous processes. Aggregation and fragmentation processes will be included later. 
+
+One-dimensional (1D) PBM
```

Comment:
I don't think we need the differentiation between 1D and 2D here since they are handled by the unit operation. Focus on the population balance equations.

Reply:
While I generally agree, the terminology of internal and external coordinates when combining the PBM module with STR or PFR modules should be introduced.